### ISSUE TITLE
Can't install on R version 3.4.

### ISSUE THREAD

**Issue by GautamBuddha** [ipa-tys/ROCR#7]
I get this error message: 
"Warning in install.packages :
  package ‘ROCR’ is not available (for R version 3.4.3)"

I am using RStudio Version 1.1.423 on Macintosh; Intel Mac OS X 10_15_4.

**Comment by FelixErnst** [comment]
Please provide the exact script you are using for trying to install `ROCR`. Also provide the output of `sessionInfo()`

edit: The message is not generated by `ROCR`, probably by the `install.packages()` function. It tells you exactly the problem, that the current version of `ROCR` (`1.0-11`) is not available for `R 3.4.3`. The current version of `ROCR` requires `R >= 3.6`.

You can install either the older version by running 
```
install.packages("https://cran.r-project.org/src/contrib/Archive/ROCR/ROCR_1.0-7.tar.gz", repo = NULL, type = "source")
```

or update your R installation to `R 3.6.3` or `R 4.0`.

**Comment by GautamBuddha** [comment]
Thanks, upgrading to R 4.0 solved the problem.
